src/renderer/pages/Assets/Assets/components/AssetsList/AssetsList.tsx

```diff
@@ -38,11 +38,23 @@ export const AssetsList = () => {
   useEffect(() => {
     const isMultisig = walletUtils.isMultisig(activeWallet);
 
+    const availableChains = activeAccounts.some(accountUtils.isAllChainsSupport)
+      ? new Set(Object.keys(chains) as ChainId[])
+      : activeAccounts.reduce<Set<ChainId>>((acc, account) => {
+          const chainId = (account as ChainAccount).chainId;
+
+          if (chainId) {
+            acc.add(chainId);
+          }
+
+          return acc;
+        }, new Set<ChainId>());
```

Comment:
May I suggest to use a `filter` here for readability, something like
```suggestion
      : new Set(activeAccounts.filter((account) => !!account.chainId))
```

Reply:
Thibaut's suggestion is nice and clean, `reduce` makes no calculations inside just filtering
```javascript
new Set(activeAccounts.filter((account) => Boolean(account.chainId)))
```